doc/getting_started_linux.rst

```diff
@@ -219,3 +219,28 @@ Run a test case until the problem occurs (in the build directory)::
 Study the resulting core::
 
     gdb pf_test core
+
+SNMP (Conformance class B)
+--------------------------
+
+Conformance class B requires SNMP support. P-Net for Linux implements
+a Net-SNMP subagent that handles the Profinet mandatory MIB:s. Also
+see :ref:`network-topology-detection` for information regarding SNMP.
+
+Enable SNMP by setting PNET_OPTION_SMP to ON. Net-SNMP also needs to
+be installed. On Ubuntu you can install the required packages using:
+
+    sudo apt install -y snmpd libsnmp-dev
+
+The file snmpd.conf controls access to the snmp agent. The following
+needs to be set:
```

Comment:
Use double colon to start a code block (a single colon will be shown). (As you already have done, use blank line before and after, and indent the block).